<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaFifo</class>
 <widget class="QDialog" name="VentanaFifo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rafaga de CPU</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRafaga">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tiempo de Llegada</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLlegada">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAgregar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIniciar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblProcesos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>571</width>
     <height>141</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>#Proceso</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rafaga CPU</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tiempo de LLegada</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Estado</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Resultados</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblResultados">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>571</width>
     <height>131</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>#Proceso</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tiempo de espera</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tiempo de Sistema</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VentanaFifo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VentanaFifo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
